src/commands/coin/coinInfo.ts

```diff
@@ -0,0 +1,46 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+class CoinInfo extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'coin-info',
+      aliases: ['ci', 'coininfo', 'codeycoin info'],
+      group: 'coin',
+      memberName: 'coin-info',
+      description: 'Info about Codey Coin, including ways to get and how to spend Codey Coins.',
+      examples: [`${client.commandPrefix}coin-info`, `${client.commandPrefix}ci`, `${client.commandPrefix}coininfo`]
+    });
+  }
+  infoEmbed = new MessageEmbed()
+    .setColor(EMBED_COLOUR)
+    .setTitle('🪙 About Codey Coin 🪙')
+    .setThumbnail('') // Thumbnail, if needed?
+    .setDescription('Codey Coin is a virtual currency... (what it is, maybe some rules around it) \n')
+    .addFields(
+      {
+        name: 'How Can I Obtain Codey Coins?',
+        value:
+          'You can obtain Codey Coins by participating in Discord activities, such as:\n' +
```

Comment:
nit: you can use multiline strings if it's easier, so you don't have to keep concatenating strings

```javascript
`this is a multiline string
second line
third line
...
`
```

Reply:
Ah okay thank you, I tried doing that at first with ' ' and it wasn't showing up as a string so I gave up. Will change this, it's easier and looks better